### PR TITLE
[DNM] Get installer code from hackday

### PR DESCRIPTION
**Pull request by Kmaschta**
:x: Do not merge this PR

## TODO
- [x]   Move lambda code into a separated folder
- [x] Add installer code
- [x] Remove `frontend` folder from installer
- [ ] Clean & test installer code

### CODE REVIEW

File: app/installer/.babelrc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "env": {
+        "development": {
+            "presets": [
+                "es2015",
+                "react",
+                "react-hmre",
+                "stage-0"
```
Comment (brikou) [inline app/installer/.babelrc]: I don't think you need stage-0, as you do not use do expressions neither `::func` notation

File: app/installer/README.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
+# marmelab-boilerplate [![Build Status](https://travis-ci.org/marmelab/javascript-boilerplate.svg?branch=master)](https://travis-ci.org/marmelab/javascript-boilerplate)
```
Comment (brikou) [inline app/installer/README.md]: IMO a link to original README is enough as you don't need to maintain it yourself.

File: app/installer/config/development.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    github: {
+        token: '372e488aefdf0693ca03',
```
Comment (brikou) [inline app/installer/config/development.js]: Hello token

Comment (Kmaschta) [inline app/installer/config/development.js]: You freaking me!

**Comment by Kmaschta** [comment]
See #51

**Pull request closed without merging**